asset/styles/style.css

```diff
@@ -0,0 +1,78 @@
+/* * {
+  margin: 0;
+  padding: 0;
+} */
+footer {
+  margin-left: 2%;
+  margin-right: 2%;
+  overflow-x: hidden;
+}
+
+.header-logo {
+  height: 50px;
+}
+
+.header-main-container {
+  background-color: #f72210;
+}
+
+.btn-header {
+  border-radius: 24px;
+  height: 25px;
+  font-size: 12px;
+  text-transform: uppercase;
+  font-weight: 800;
+  text-align: center;
+}
+
+.header-menu * {
+  margin-right: 10px;
+}
+
+.fake-body {
+  height: 100vh;
+}
+
+.footer-logo img {
+  width: 60%;
+}
+
+.copyright {
+  font-size: 0.68em;
+}
+
+.flex-xs {
+  display: flex;
+}
+
+.social-icons-footer {
+  font-size: 24px;
+}
+
+.upper-footer-social-media {
+  /* width: 100%; */
+  display: flex;
+  justify-content: flex-start;
+  padding: 0;
+  position: relative;
```

Comment:
Avoid using custom styling where bootstrap styling is possible. Example in the above, you can use `d-flex`, `justify-content-start`, `p-0`, `position-relative` bootstrap classes. Please check the rest of your styling as well.

Reply:
Thanks, appreciate the review!